r-package/grf/R/average_treatment_effect.R

```diff
@@ -294,3 +295,39 @@ average_treatment_effect = function(forest,
   tau.se <- sqrt(sigma2.hat)
   return(c(estimate=tau.avg, std.err=tau.se))
 }
+
+
+#' Estimate average outcomes conditional on X and W given a trained causal forest
```

Comment:
Could you also add the definitions of Y.hat.0 and Y.hat.1 to the documentation?

Reply:
I modified the documentation, but please let me know if you'd like something different here.